### ISSUE TITLE
Mirror socrates repository

### ISSUE THREAD

**Issue by penguian** [ACCESS-NRI/accessdev-Trac-archive#185]
`keyword_TIWG` `resolution_fixed`   |    by saw562@nci.org.au

___

The UM radiance has been moved to a new repository on code.metoffice.gov.uk

http://collab.metoffice.gov.uk/twiki/bin/viewfile/Static/News/met-office/um-collab/um/general/msg00422.html

This needs to be added to the mirror & FCM configuration

___
_Issue migrated from [trac:185](https://accessdev.nci.org.au/trac/185) at 2024-01-31 18:12:31 +1100_

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _commented_


___
**Branch**: [/log/puppet?rev=saw562/socrates saw562/socrates]

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _changed status from `new` to `assigned`_

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _changed owner from `saw562` to `mrd599`_

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _commented_


___
This is now working on Accessdev-test. I've redone the FCM configuration file so that new repositories can now be added by editing `hieradata/project.yaml`.

Some keywords are automatically set up by fcm (e.g. `fcm:um_tr` and `fcm:um_br`) so they've not been added to the config file. I've also not specified version keywords since FCM can get those from the repository properties.

The extract stage of `rose stem` works fine both for the Socrates branch in um:#464 and trunk.

Martin, can you please take a look before I apply this to accessdev?

Thanks, Scott

**Comment by penguian** [comment]
_@martin.dix@anu.edu.au_ _commented_


___
fcm working ok for me.

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _changed status from `assigned` to `accepted`_

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _changed owner from `mrd599` to `saw562`_

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _commented_


___
Ok, I'll send out a notice to the mail list and install it.

**Comment by penguian** [comment]
_Scott Wales <scott.wales@unimelb.edu.au>_ _changed status from `accepted` to `closed`_

**Comment by penguian** [comment]
_Scott Wales <scott.wales@unimelb.edu.au>_ _changed resolution from `` to `fixed`_

**Comment by penguian** [comment]
_Scott Wales <scott.wales@unimelb.edu.au>_ _committed [0]_
___
In [changeset:"0c2ad1ca1ca492308d15cad820d4e244581e9545/puppet"]:
```
#CommitTicketReference repository="puppet" revision="0c2ad1ca1ca492308d15cad820d4e244581e9545"
Fix #185: Simplify FCM config, add Socrates

Merge branch 'saw562/socrates'
```
